### DISCUSSION TITLE
Help with making a clock with DS1302 module

### DISCUSSION THREAD

**Discussion by syvvc** [Makuna/Rtc#175] [General]
I am doing a school project where I am designing and coding a clock with no coding or programming experience at all. I am deciding whether to use the NodeMCU or an Arduino Nano. I decided not to use an arduino uno as the clock needs to fit in a shell which is going to be quite small. I have been following a tutorial.
 https://www.instructables.com/Real-Time-Clock-Module-With-Arduino/

Questions:
What board should I use, arduino nano or NodeMCU.
How do I use the Makuna/RTc code, do I just have to put the "#include <RtcDS1302.h>" line in and then hit verify.

<img width="1440" alt="Screen Shot 2023-05-12 at 8 38 34 am" src="https://github.com/Makuna/Rtc/assets/133199872/6262909b-53d6-4d1e-82ff-01ffba35107f">

**Comment by Makuna** [comment]
Start with a library example.  Examples, Library, Rtc By Makuna, Ds1302_Simple.  That should get you started.
